components/proxy/src/main/java/com/hotels/styx/StyxServer.java

```diff
@@ -297,6 +298,7 @@ private static StartupConfig parseStartupConfig(String[] args) {
 
     @Override
     protected void doStart() {
+        RequestDebugger.add(components.environment().configuration().get("debug-request-id-prefix").orElse(null));
```

Comment:
Should we remove this before merging?

I'm not sure how this would work because styx itself allocates a request ID and we should treat it as a pseudo random string.